src/Ray.php

```diff
@@ -17,6 +17,11 @@ public static function bootForWordPress()
         static::$queryLogger = new QueryLogger();
 
         static::$mailLogger = new MailLogger();
+
+        // Attempt to turn on SAVEQUERIES which gets us more detailed query logging.
+        if (!defined('SAVEQUERIES')) {
```

Comment:
Why already turn it on here? We don't know yet if users do want to log queries.

Reply:
Yeah, that's a fair point.